app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -189,6 +190,9 @@ const AvailableActions = createReactClass({
       controls.push((
         <p key="download_course_stats"><CourseStatsDownloadModal course={course} /></p>
       ));
+      controls.push((
+        <EmbedStatsButton title={course.title} key="embed_course_stats"/>
```

Comment:
Needs to be wrapped in a `<p>` to the margin is right:

![button margin](https://user-images.githubusercontent.com/848483/49613957-c9efa280-f95d-11e8-9c19-e96ceb57179a.png)